general/available_hardware.md

```diff
@@ -1,63 +1,77 @@
-# Laptops
+# List of Available Hardware
 
-## Apple 13-inch MacBook Pro - Space Gray 
-- 2.3GHz dual-core Intel Core i5 processor, Turbo Boost up to 3.6GHz
+## Laptops
+
+### Apple 13-inch MacBook Pro - Space Gray
+
+- 2.3GHz quad-core 8th-generation Intel Core i5 processor, Turbo Boost up to 3.8GHz
 - 16GB 2133MHz LPDDR3 memory
 - 256GB PCIe-based SSD
-- Intel Iris Graphics 640
-- Two Thunderbolt 3 ports
+- Intel Iris Graphics 655
+- Four Thunderbolt 3 (USB-C) ports
 - Force Touch trackpad
+- Touch Bar and Touch ID
```

Comment:
This is not the base one, remove this

Reply:
Indeed, yet the base model is very outdated. Please take the time to read https://github.com/src-d/guide/issues/310 and understand the reasoning behind it and why it's a good decision.